go.mod

```diff
@@ -274,7 +274,9 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101503.2-rc.1
+//replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101503.2-rc.1
+
+replace github.com/ethereum/go-ethereum => github.com/leruaa/op-geth v1.0.0-rc.1
```

Comment:
at which commit did you tag `github.com/leruaa/op-geth` `v1.0.0-rc.1`? at https://github.com/ethereum-optimism/op-geth/pull/551/commits/e8101bbc1ada202f985b7a3c34a2e25eb4068b01 ?

Reply:
yes `e8101bbc1ada202f985b7a3c34a2e25eb4068b01`